src/test/resources/testdb/schema.sql

```diff
@@ -1,22 +1,37 @@
+Drop table if exists reply;
+Drop table if exists article;
 Drop table if exists member;
 
 CREATE TABLE member
 (
-    id           bigint AUTO_INCREMENT primary key,
-    user_name    varchar(255) not null,
-    password     varchar(255) not null,
-    email        varchar(255) not null,
-    created_date timestamp default now()
+    id       bigint AUTO_INCREMENT primary key,
+    userName varchar(64) not null unique,
+    password varchar(64) not null,
+    nickName varchar(64) not null,
+    email    varchar(64) not null
 );
 
-Drop table if exists article;
-
 CREATE TABLE article
 (
     id           bigint AUTO_INCREMENT primary key,
-    writer       varchar(255),
+    member_id    bigint,
+    writer       varchar(64),
     title        varchar(255),
     content      text,
     created_date timestamp default now(),
-    reply_count  integer   default 0
+    reply_count  integer   default 0,
+    FOREIGN KEY (member_id) REFERENCES member (id)
+);
+
+CREATE TABLE reply
+(
+    id         bigint AUTO_INCREMENT primary key,
```

Comment:
```suggestion
    id         bigint PRIMARY KEY AUTO_INCREMENT,
```
AUTO INCREMENT보다 PRIMARY KEY가 더 중요도가 높은 구문이라 앞에 두는 게 좋습니다.